style.css

```diff
@@ -0,0 +1,4 @@
+h1 {
```

Comment:
- [ ] Kindly make sure you use class a recommended by the project requirements [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header) :recycle: